pylint/pyreverse/main.py

```diff
@@ -191,6 +191,21 @@
             help="set the output directory path.",
         ),
     ),
+    (
+        "disable-path-patching",
+        dict(
```

Comment:
Please use `{}` instead of `dict`.

Reply:
I used dict because all of the options in pyreverse also used dict. I can {} if it's fine to mismatch the rest of the file.